xbmc/pictures/Picture.cpp

```diff
@@ -55,6 +56,27 @@ bool CPicture::GetThumbnailFromSurface(const unsigned char* buffer, int width, i
   return true;
 }
 
+bool CPicture::CreateBase64ThumbnailFromSurface(const unsigned char *buffer, int width, int height, int stride, const std::string &thumbFile, std::string &base64buffer)
+{
+  CLog::Log(LOGDEBUG, "cached image '%s' size %dx%d", CURL::GetRedacted(thumbFile).c_str(), width, height);
+  
+  unsigned char *thumb = NULL;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

use nullptr